activestorage/test/test_helper.rb

```diff
@@ -85,6 +86,19 @@ def extract_metadata_from(blob)
     def fixture_file_upload(filename)
       Rack::Test::UploadedFile.new file_fixture(filename).to_s
     end
+
+    def append_service
```

Comment:
I think this is better named `with_service` and make the configured `service` injectable via an argument. That was it is clearer in the tests where the service configuration is coming from.

Reply:
agreed, changed it (and added couple more similar methods)